src/helpers.js

```diff
@@ -49,7 +53,9 @@ function mergeHashes(source, target = {}, old, keepRemoved = false) {
         keepRemoved
       )
       target[key] = nested.new
-      old[key] = nested.old
+      if (Object.keys(nested.old).length > 0) {
```

Comment:
This project favours this format: `Object.keys(nested.old).length` or use `_.isEmpty` since it is already loaded.